deploy/olm-catalog/akka-cluster-operator/0.2.3/akka-cluster-operator-certified.v0.2.3.clusterserviceversion.yaml

```diff
@@ -170,7 +170,7 @@ spec:
                       - name: OPERATOR_NAME
                         value: akka-cluster-operator
                       - name: RELATED_IMAGE_CONTROLLER
-                        value: registry.connect.redhat.com/lightbend/akkacluster-operator:v0.2.2   
+                        value: registry.connect.redhat.com/lightbend/akka-cluster-operator-certified:0.2.2
```

Comment:
And this 0.2.3?

Reply:
This was likely a metadata only release. I recall the driver of the change being the metadata (yaml), so I pushed a new CSV w/out a matching 0.2.3 container. Sub-optimal, I admit.